gulpfile.js

```diff
@@ -148,7 +148,9 @@ gulp.task('installFixtures', function() {
     }, 1 * 1000);
     shell.cd('test/fixtures');
 
-    execAsync('npm install --quiet', {cwd: '../fixtures'}).then(() => {
+    execAsync('type yarn &> /dev/null | yarn install || npm install --quiet', {
```

Comment:
This worked for me:

```js
yarn --version >nul 2>&1 && ( yarn install ) || ( npm install --quiet )
```

We should be able to use [process.platform](https://nodejs.org/api/process.html#process_process_platform) to figure out what the user is on.